aten/src/ATen/native/quantized/cudnn/Linear.cpp

```diff
@@ -69,11 +69,11 @@ void PackedLinearWeightCudnn::apply_impl_helper(const at::Tensor& quantized_outp
   c10::optional<at::Tensor> bias_multiplier_tensor;
   c10::optional<at::Tensor> broadcasted_bias;
   if (bias_.has_value()) {
-    // the input bias is a 1-D tensor whose size is the same as the size of the second dimension of quantized_output.
+    // the input bias is a 1-D tensor whose size is the same as the size of the last dimension of quantized_output
     // we need to add trailing dimensions in order to properly broadcast bias, otherwise broadcast_to will fail.
     // the number of trailling dimensions is quantized_output.dim() - 2. We also prepend a leading dimension for clarity
     std::vector<int64_t> new_size(quantized_output.dim(), 1);
-    new_size[1] = bias_.value().size(0);
+    new_size.back() = bias_.value().size(0);
```

Comment:
do we need to change the size of `new_size` now? can it be the same size as bias?

Reply:
I need to check if broadcast addition works in cudnn 8.4.0, if it does, then we don't need new_size

Reply:
@jerryzh168  Channel broadcasting addition seems to work, but both conv and linear are running into that `uid` issue I mentioned on slack. I think maybe we should keep the old format (keep `new_size` for now) until NVIDIA figures out what the issue is? Or I could implement it acknowledging the `uid` issue (it's not blocking, but we will need frontend `v0.6` for it because the buffer size for the previous frontend is fixed at 10. we need 11 right now due to the problem I identified)

Reply:
I see, can we try upgrading to `v0.6`? I saw it is reverted, what is the plan now?

Reply:
@jerryzh168 It looks like cudnn updated the frontend to address the return statement error: https://github.com/NVIDIA/cudnn-frontend/commit/fa611998a360cbabaa2dcc7c9859748144114fc0

I'll try re-landing the PR

Reply:
btw why do we change from new_sizes[1] to new_sizes.back() here? could you explain in PR summary?